### DISCUSSION TITLE
Sunbird Knowledge platform local installation

### DISCUSSION THREAD

**Discussion by yogeshsarmalkar-trigyn** [Sunbird-Ed/Community#384] [Installation] [answered]
Hi Team,
1. Is is necessary to install docker with all the databases (neo4j, Redis, Cassandra ) in the local machine for knowledge platform https://github.com/project-sunbird/knowledge-platform/tree/release-5.1.0 ? 
2. Can we configure https://ed.sunbird.org/use-1/install-locally/sunbirded-portal to point it to the local knowledge-platform and test the API flow for content service ? If yes, let me know the configuration file which will help me to point it to local content service ?

**Comment by vinukumar-vs** [comment, marked as answer]
@yogeshsarmalkar-trigyn This we didn't tried as part of local development of portal. If you configure the portal backend to point to local knowlg services. Please refer the wiki to point to local knowlg services


>  https://github.com/Sunbird-Ed/SunbirdEd-portal/tree/release-5.1.0#project-setup
> Edit the Application Configuration

@rajeevsathish @Krishnaj20

**Reply by Krishnaj20** [reply]
@yogeshsarmalkar-trigyn - is your query resolved?

@gandham-santhosh @kameshbhr @anandvarada @vinukumar-vs

**Reply by yogeshsarmalkar-trigyn** [reply]
@Krishnaj20 I am trying to connect to the local knowledge services, but yes we can close this thread. 
I will raise a new ticket if I face any further issues
